main.go

```diff
@@ -210,48 +210,33 @@ func convertField(curPkg *ProtoPackage, desc *descriptor.FieldDescriptorProto, m
 
         fieldOptions := desc.GetOptions()
         if fieldOptions != nil {
-          // is there is a 'required' annotation value on this field?
-          if proto.HasExtension(fieldOptions, protos.E_Require) {
-            requireValue, err := proto.GetExtension(fieldOptions, protos.E_Require)
-            if err == nil {
-              doRequire := *requireValue.(*bool)
-              if (doRequire) {
-                // annotations are telling us to force-require this field, so do it
-                field.Mode = "REQUIRED"
-              }
-            }
-          }
+          // try to resolve BQ field options
+          if proto.HasExtension(fieldOptions, protos.E_BigQuery) {
+            _rawOptions, err := proto.GetExtension(fieldOptions, protos.E_BigQuery)
```

Comment:
refactored `main.go` to check at `protos.E_BigQuery`, type cast once, and check against that message